src/command_modules/azure-cli-storage/azure/cli/command_modules/storage/tests/latest/test_storage_blob_scenarios.py

```diff
@@ -131,6 +131,37 @@ def is_block_put_req(request):
                          'The expected number of block put requests is {} but the actual '
                          'number is {}.'.format(block_count, len(put_blocks)))
 
+    @ResourceGroupPreparer()
+    @StorageAccountPreparer()
+    def test_storage_blob_socket_timeout(self, resource_group, storage_account):
+        local_dir = self.create_temp_dir()
+        local_file = self.create_temp_file(1)
+        blob_name = self.create_random_name(prefix='blob', length=24)
+        account_info = self.get_account_info(resource_group, storage_account)
+
+        container = self.create_container(account_info)
+
+        with self.assertRaises(Exception):
```

Comment:
`CLIException` instead of `Exception`. Narrow down the assertion to make sure real issue is exposed.